lib/shared/pe_status_check.rb

```diff
@@ -153,6 +155,28 @@ def postgres?
       service_file_exist?('pe-pgsql/pe-postgresql')
   end
 
+  # Get the maximum defined and current concurrent connections to Postgres
+  def self.psql_return_result(sql, psql_options = '')
+    command = %(su pe-postgres --shell /bin/bash --command "cd /tmp && #{PUP_PATHS[:server_bin]}/psql #{psql_options} --command \\"#{sql}\\"")
```

Comment:
#{PUP_PATHS[:server_bin]} is never declared anywhere so this will not resolve correctly